src/main/java/com/glodblock/github/common/tile/TileLevelMaintainer.java

```diff
@@ -165,7 +174,8 @@ private TickRateModulation doWork() {
                     if (!isCraftable) requests.updateState(i, State.Error);
                     if (craftingGrid.canEmitFor(craftItem) || craftingGrid.isRequesting(craftItem)
                             || !isDone
-                            || !shouldCraft)
+                            || !shouldCraft
+                            || allBusy)
```

Comment:
Minor nitpick: I would reorder these checks so that boolean variables are tested first, and only then methods on `craftingGrid`. This would further save a trivial amount of performance using short-circuiting.

Reply:
Sure, done.